modules/platform/nuxeo-platform-video/nuxeo-platform-video-core/src/test/java/org/nuxeo/ecm/platform/video/extension/TestVideoImporterAndListeners.java

```diff
@@ -115,20 +114,20 @@ protected static Blob getBlobFromPath(String path, String mimeType) throws IOExc
 
     @Test
     public void testVideoType() {
-        DocumentType videoType = session.getDocumentType(VIDEO_TYPE);
+        DocumentType videoType = session.getDocumentType(VideoConstants.VIDEO_TYPE);
```

Comment:
yes it's what i did in the past until the `Glacier` developments that I did where Florent asked me to avoid the  static import and use them really for the known things (like StringUtils for example). His idea is to simply the class reading. :)

but if we all agree i will update it

Reply:
I took your suggestion about the static import, but i keep the conversion open for more feedbacks.

Reply:
same suggestion than [here](https://github.com/nuxeo/nuxeo/pull/3884#discussion_r402907627)

Reply:
When I mentioned avoiding static imports I was talking mostly about methods, not constants.